mlflow/server/js/src/experiment-tracking/components/ParallelCoordinatesPlotPanel.js

```diff
@@ -25,36 +25,126 @@ export class ParallelCoordinatesPlotPanel extends React.Component {
     sharedParamKeys: PropTypes.arrayOf(String).isRequired,
     // An array of metric keys shared by all runs
     sharedMetricKeys: PropTypes.arrayOf(String).isRequired,
+    // An array of different parameter keys across runs
```

Comment:
I was a little confused by this, since it seems to be in opposition of "sharedParamKeys", but it doesn't mean param keys which don't exist in some runs. Maybe this should say

"A subset of sharedParamKeys where the values of the parameters differ between runs."

Reply:
That is better. I'll fix the comment.